python/test_component.py

```diff
@@ -0,0 +1,16 @@
+import sys
+from v9 import V9Component
+
+
+def test(http_method, path, request_arguments, request_body):
+    print(http_method, path, request_arguments, request_body)
+    return 200, "all good"
+
+
+if __name__ == '__main__':
+    print("Arguments " + str(sys.argv))
+    comp = V9Component(sys.argv[1], sys.argv[2])
+
+    comp.register_operation("Test", test)
```

Comment:
Can a component have multiple operations?

Reply:
Yes, you just need to call `register_operation` multiple times. For simplicity the example component just has one -- should I add another?

Reply:
No need to add another, just wanted to confirm that a component can support multiple functions.

Reply:
Also as a note, I plan to add opt-in automatic JSON desearlizing/serializing in the next draft